docs/changelog/2003.feature.rst

```diff
@@ -0,0 +1,3 @@
+Optionally skip VCS ignore directive for entire virtualenv
+directory, using command line option :option:`no-vcsignore`.
```

Comment:
Please drop the ``command line`` words - as https://virtualenv.pypa.io/en/latest/cli_interface.html#defaults outlines this is just an option that's configurable from either env var, config file, or cli flag.